Source/UI/IdentityUIConfiguration.swift

```diff
@@ -60,6 +60,13 @@ public struct IdentityUIConfiguration {
         }
     }
 
+    public var appVersion: String {
+        guard let name = Bundle.main.infoDictionary?[kCFBundleVersionKey as String] as? String else {
+            preconditionFailure("Could not fetch bundle name. Please set IdentityUIConfiguration.appName")
```

Comment:
Isn't this supposed to be something that much be changed in the app bundle and not the IdentityUIConfig object?

Reply:
this is not supposed to be changed at all.

Reply:
The error message is off then isn't it?